src/components/shared/chart/Viewport.js

```diff
@@ -388,14 +425,22 @@ export const withChartViewport: WithChartViewport<*, *> =
         const mouseX = event.clientX;
         const { containerLeft, containerWidth } = this.state;
         const innerContainerWidth = containerWidth - marginLeft - marginRight;
+        const mouseCenter =
+          (mouseX - containerLeft - marginLeft) / innerContainerWidth;
+        this.zoomRangeSelection(mouseCenter, deltaY);
+      }
 
-        const { maximumZoom } = this.props.viewportProps;
+      zoomRangeSelection = (center, deltaY) => {
```

Comment:
Since it is no longer only the mouse which will trigger this, `deltaY` should be renamed.